.github/workflows/php-changes-detection.yml

```
@@ -20,7 +20,7 @@ jobs:
               uses: tj-actions/changed-files@0874344d6ebbaa00a27da73276ae7162fadcaf69 # v44.3.0
               with:
                   files: |
-                      **.{php}
+                      **.php
```

```suggestion
                      '**.php'
```

The [filter pattern cheatsheet](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet) advises this be wrapped in quotes to avoid a parse error:
> If you start a pattern with `*`, `[`, or `!`, you must enclose the pattern in quotes.

(Though it doesn't appear to have caused the workflow to fail in this case.)